layouts/_default/baseof.html

```diff
@@ -0,0 +1,30 @@
+<!-- 
+  Copyright (c) 2018 Eclipse Foundation, Inc.
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+  Contributors:
+    Christopher Guindon <chris.guindon@eclipse-foundation.org>
+  SPDX-License-Identifier: EPL-2.0
+-->
+<!DOCTYPE html>
+<html lang="{{ .Lang }}">
+<head>
+  {{ block "head" . }}{{ partial "head.html" . }}{{ end }}
+  {{ block "head-custom" . }}{{ partial "head_custom.html" . }}{{ end }}
+  {{ block "head-suffix" . }}{{ partial "head_suffix.html" . }}{{ end }}
+</head>
+<body>
+  {{ block "header" . }}{{ partial "header.html" . }}{{ end }}
+  {{ block "breadcrumbs" . }}{{ if ne .Page.Params.hide_breadcrumb true }}{{ partial "breadcrumbs.html" . }}{{end}}{{ end }}
+  {{ block "featured_story" . }}{{ partial "featured_story.html" . }}{{ end }}
+  {{ block "mobile-sidebar" . }}{{ partial "mobile-sidebar.html" . }}{{ end }}
```

Comment:
Instead of creating a new baseof.html file, you could maybe add the mobile-sidebar in main_prefix.html just like we do for the regular sidebar in main_suffix.html.

Reply:
Updated in main_prefix